pkgs/top-level/all-packages.nix

```
@@ -7683,6 +7683,8 @@ with pkgs;
 
   edk2-uefi-shell = callPackage ../tools/misc/edk2-uefi-shell { };
 
+  lanzaboote-uefi-stub = callPackage ../os-specific/uefi/lanzaboote { };
```

The attr is lanzaboote-uefi-stub, the directory just lanzaboote and the pname lanzaboote-stub. Can we reduce that to just two?

```suggestion
  lanzaboote-uefi-stub = callPackage ../os-specific/uefi/lanzaboote-stub { };
```